hledger/Hledger/Cli/Commands/Balance.hs

```diff
@@ -388,11 +388,11 @@ balance opts@CliOpts{reportspec_=rspec} j = case reporttype_ of
 balanceReportAsCsv :: ReportOpts -> BalanceReport -> CSV
 balanceReportAsCsv opts (items, total) =
   ["account","balance"] :
-  [[accountNameDrop (drop_ opts) a, wbToText $ showMixedAmountB oneLine b] | (a, _, _, b) <- items]
+  [[accountNameDrop (drop_ opts) a, wbToText $ showMixedAmountB (balanceOpts False opts) b] | (a, _, _, b) <- items]
```

Comment:
Could you say a little more about the changes in this file ? What drove them ?

Reply:
An attempt to reduce code duplication and make sure the display opts used in different parts of the balance report don't get out of sync. I'm inclined to leave this one in, but let me know if you disagree.